packages/ffe-form-react/src/BaseRadioButton.js

```diff
@@ -48,8 +50,9 @@ class BaseRadioButton extends Component {
                         'ffe-radio-input--dark': dark,
                     })}
                     id={this.id}
+                    ref={innerRef}
                     type="radio"
-                    checked={isSelected}
+                    defaultChecked={isSelected}
```

Comment:
This feels very wrong. The component will re-render when its props change, wether `checked` or `defaultChecked` is used on a child should have no effect on that?

Reply:
I pasted this into the Checkbox.md locally and used `npm start` to run the designsystem docs

```
initialState = { checked: true };
<>
    <fieldset className="ffe-fieldset">
        <legend className="ffe-form-label">Hva er du interessert i?</legend>
        <Checkbox name="interests" value="sport" inline={false} defaultChecked={state.checked}>
            Sport
        </Checkbox>
        <Checkbox name="interests" value="food" inline={false}>
            Matlaging
        </Checkbox>
        <Checkbox name="interests" value="moon" inline={false}>
            Månen
        </Checkbox>
    </fieldset>
    <button onClick={() => setState({ checked: !state.checked})}>swap</button>
</>
```

Using `defaultChecked`, the checkbox does not update when the state changes. Using `checked` it does. So `defaultChecked` is only used during the initial render like the documentation says. If it works in your usecase you're most likely remounting the component on every change. You can check that using `useEffect` or `componentDidMount`.

Reply:
I can see how it would break things. I've tried a different approach, that leaves the checked attribute undefined when selectedValue is undefined, leaving state management to the browser.